contributing_to_docs/doc_guidelines.adoc

```diff
@@ -187,7 +187,9 @@ If you create a directory with a multiple-word name, separate each word with an
 Do not italicize user-replaced values. This guideline is an exception to the link:https://redhat-documentation.github.io/supplementary-style-guide/#user-replaced-values[Red Hat supplementary style guide].
 ====
 
-Do not create or rename a top-level directory in the repository and topic map without checking with the docs program manager first. In the main OpenShift docs, you can create one level of subdirectories. In the docs for features that are designed to be used with OpenShift, such as Service Mesh and OpenShift Virtualization, you can create two levels of subdirectories.
+Do not create or rename a top-level directory in the repository and topic map without checking with the docs program manager first.
+
+Avoid creating two levels of subdirectories, because the link:https://github.com/openshift/openshift-docs/issues/52149[breadcrumb bar on docs.openshift.com breaks]. If you have a valid use case for two levels of subdirectories, talk with your DPM/CS for approval before creating it.
```

Comment:
```suggestion
Avoid creating two levels of subdirectories because the link:https://github.com/openshift/openshift-docs/issues/52149[breadcrumb bar on docs.openshift.com breaks]. If you have a valid use case for two levels of subdirectories, talk with your DPM/CS (and, for aligned teams, the OpenShift DPM) for approval before creating it.
```

Reply:
@kalexand-rh updated, let me know if it's good now or if you have any more feedback. thanks!

Reply:
You still don't need that comma, but it otherwise LGTM.

Reply:
Oop didn't see that. Will remove!